BSIP 88: Increase maximum total Bitshares supply to grow the reserve pool fund

grctest opened the pull request:
The concept of this BSIP was introduced 2 weeks ago:
https://github.com/bitshares/bsips/issues/295
https://bitsharestalk.org/index.php?topic=34038.0

If you have any input on this idea, please do comment in this pull request, in the issue and even in bitsharestalk.

grctest committed:
Create bsip-0088.md

brekyrself commented:
This should move forward as soon as possible so it can be voted on by bts holders as indicated in the bsip.  BTS needs development funds and this is a very democratic way to jumpstart new workers.

"Create a worker proposal to evaluate whether people are for or against such an increase in core token supply for reserve pool purposes.

If it passes, create secondary worker proposals to vote on the amount to increase the total supply by."

grctest committed:
Update bsip-0088.md

grctest commented:
I've updated the BSIP with some simplifications, removed discussions which didn't invoke comment, and added a second option to the proposed polling mechanism.

I'm feeling more in favour of option B - just voting on whether or not to increase the maximum total supply value rather than voting on the intent, then voting on the specific amount.

Discussing the amount in here could suffice too - 100 Billion BTS up from ~3.6 Billion BTS - IMO with such a change there would be no need for further maximum total supply increases probably for decades to come.

Since there's the refund400k reserve pool barrier to entry, it doesn't really matter what the increased amount is after a certain point, but the amount we increase it by can send a powerful message.
